README.md

```diff
@@ -1,117 +1,119 @@
+| Coding Style | [coding-style.md](/policy/coding-style.md) |
+| Best Practices | [coding-best-practices.md](/policy/coding-best-practices.md) |
+| Review Process | [review-process.md](/policy/review-process.md) |
+| Guidelines for building the driver | [BUILDGUIDE.md](BUILDGUIDE.md) |
+| Guidelines for Contributors | [CONTRIBUTING.md](CONTRIBUTING.md) |
+| Changelog for all driver releases | [CHANGELOG.md](CHANGELOG.md) |
+| Support Policy | [SUPPORT.md](SUPPORT.md) |
+| Code of Conduct | [CODE_OF_CONDUCT.md](CODE_OF_CONDUCT.md) |
+| Copyright Information | [COPYRIGHT.md](COPYRIGHT.md) |
+
+## Our Featured Contributors
+
+Special thanks to everyone who has contributed to the project.
+We thank you for your continuous support in improving the SqlClient library!
+
+- Edward Neal ([@edwardneal](https://github.com/edwardneal))
+- Erik Ejlskov Jensen ([@ErikEJ](https://github.com/ErikEJ))
+- Michel Zehnder ([@MichelZ](https://github.com/MichelZ))
+- Shay Rojansky ([@roji](https://github.com/roji))
+- Phillip Haydon ([@phillip-haydon](https://github.com/phillip-haydon))
+- Rasmus Melchior Jacobsen ([@rmja](https://github.com/rmja))
+- Robin Sue ([@Suchiman](https://github.com/Suchiman))
+- Simon Cropp ([@SimonCropp](https://github.com/SimonCropp))
+- Stefán Jökull Sigurðarson ([@stebet](https://github.com/stebet))
+- Stephen Toub ([@stephentoub](https://github.com/stephentoub))
+- Wraith ([@Wraith2](https://github.com/Wraith2))
+
+Up-to-date list of contributors: [Contributor Insights](https://github.com/dotnet/SqlClient/graphs/contributors)
+
+## Release Notes
+
+All preview and stable driver release notes are available under [release-notes](release-notes).
+
+## Porting from System.Data.SqlClient
+
+Refer to [porting-cheat-sheet.md](porting-cheat-sheet.md) for a safe porting experience from System.Data.SqlClient to Microsoft.Data.SqlClient and share your experience with us by enhancing this guide for future developers.
+
+## Still have questions?
+
+Check out our [FAQ](https://github.com/dotnet/SqlClient/wiki/Frequently-Asked-Questions). Still not answered? Create an [issue](https://github.com/dotnet/SqlClient/issues/new/choose) to ask a question.
+
+<!-- BEGIN MICROSOFT SECURITY.MD V0.0.3 BLOCK -->
+
+## Security
+
+Microsoft takes the security of our software products and services seriously, which includes all source code repositories managed through our GitHub organizations, which include [Microsoft](https://github.com/Microsoft), [Azure](https://github.com/Azure), [DotNet](https://github.com/dotnet), [AspNet](https://github.com/aspnet), [Xamarin](https://github.com/xamarin), and [our GitHub organizations](https://opensource.microsoft.com/).
+
+If you believe you have found a security vulnerability in any Microsoft-owned repository that meets Microsoft's [Microsoft's definition of a security vulnerability](https://docs.microsoft.com/en-us/previous-versions/tn-archive/cc751383(v=technet.10)), please report it to us as described below.
```

Comment:
```suggestion
If you believe you have found a security vulnerability in any Microsoft-owned repository that meets [Microsoft's definition of a security vulnerability](https://www.microsoft.com/en-us/msrc/definition-of-a-security-vulnerability), please report it to us as described below.
```

Repeated word. Also updated the link as per the MSRC [homepage](https://msrc.microsoft.com/).